### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by kacperd**
Update readme adding section about installation and short usage example.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,6 +21,29 @@ Optionally:
...
+- **CocoaPods:** - add `pod 'kevin-ios'` to your `Podfile`
+
+## Usage example
+
+### Initiate payment
+
+```
+import Kevin
+
+let paymentID = # your payment ID
```
Comment (PocketDaniel) [inline README.md:35]: Just a nitpick
```suggestion
let paymentID = "UUID" // Put your actual payment's ID here
```

File: README.md [hunk 2]
```diff
@@ -21,6 +21,29 @@ Optionally:
...
+
+let configuration = try KevinPaymentSessionConfiguration.Builder(paymentId: paymentID)
+    .setPaymentType(.bank)
+    .build()
+
+KevinPaymentSession.shared.delegate = self #  Set the delegate to obtain payment  session result
```
Comment (PocketDaniel) [inline README.md:41]: I think we should make it copy-paste safe? 
```suggestion
KevinPaymentSession.shared.delegate = self // Set the delegate to obtain payment session result
```

Comment (PocketDaniel) [inline README.md:41]: Also, maybe we should add at least one delegate method here to show case it?

**Review by PocketDaniel — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (kacperd) [reply README.md]: I wanted to keep the usage sample to a minimum but yeah, it probably makes sense to also include delegate code 👌

**Review by typuk — APPROVED** [review]

**Pull request merged**